README.md

```diff
@@ -0,0 +1,168 @@
+# Air Pollution metric
+
+Mobile version of SPA that indicates the levels of pollution in the air in the countries of Europe (more continents will be added eventually) - React-Redux was used for the creation and ran Test using Jest.
+
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h3 id="title"><b>SPA website</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 Air Pollution metric <a name="about-project"></a>
+
+Discover the current levels of pollution in the air with this mobile website, you can see a graph indicating the distribution of the elements present in the region!
+
+## 🛠 Built With <a name="built-with"></a>
+
+- HTML
+- CSS
+- JS
+- React
+- Redux
+- Jest
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/html/">HTML</a></li>
+    <li><a href="https://www.w3schools.com/css/">CSS</a></li> 
+    <li><a href="https://create-react-app.dev/docs/getting-started">React-app</a></li> 
+    <li><a href="https://www.w3schools.com/js/">Javascript</a></li> 
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+1 Fetch API for pollution data<br>
+2 Graph for pollution values<br>
+3 Search bar<br>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"> here is Love Demo</a>
+
+- [Live Demo](https://air-pollution-app.onrender.com)
+- Project Loom video explaination [here](https://www.loom.com/share/34264643b5a342cf85d86c689939ab2a)
+
```

Comment:
- [ ] The video presentation only shows the following image and none of the actual running project. Kindly make sure to record the app running while presenting it and also please don't forget to mention a feature or a piece of code that you're proud of.

![image](https://github.com/kazim110/AirPollutionApp/assets/62290545/27f6c00c-961a-4a0f-a82b-b1b3bfb1985b)